don't depend on py3-setuptools, and also import the system data dir
--- a/bindings/python/__init__.py
+++ b/bindings/python/__init__.py
@@ -1,5 +1,3 @@
-from pkg_resources import resource_filename
-
 from .verovio import *
 
-verovio.setDefaultResourcePath(resource_filename('verovio', 'data'))
\ No newline at end of file
+verovio.setDefaultResourcePath('/usr/share/verovio')
